arch/x86/zefi/zefi.c

```diff
@@ -123,7 +123,8 @@ uintptr_t __abi efi_entry(void *img_handle, struct efi_system_table *sys_tab)
 	for (volatile int i = 0; i < 50000000; i++) {
 	}
 
-	__asm__ volatile("cli; jmp *%0" :: "r"(code));
+	__asm__ volatile("cli; movq %0, %%rbx; jmp *%1"
+			 :: "r"(NULL), "r"(code) : );
```

Comment:
Property, this needs a clobber declaration for RBX.   Obviously it's not a problem in practice because we're about to return (and the volatile asm block means nothing will reorder after us).  But still nice to be futureproof.